src/modules/file-upload-input.marko

```diff
@@ -0,0 +1,84 @@
+<div id="file-upload-input">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-upload-input allows users to upload one or more files to a location.
+    </p>
+    <h4>Drag and Drop</h4>
+    <p>
+        Since a label element is used to achieve the UI aesthetic, the drag and drop interaction will need to be added with JS.
+    </p>
+
+    <h3>Base</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="file-upload-input">
+                <input
```

Comment:
Do we need the `multiple` property? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#multiple

Reply:
Oh good point, should we add by default so that it would support both single or multiple upload?

Reply:
Hmm, I'd always assumed we'd want multiple. If not, perhaps we just call it out in the docs that `multiple` property can be added or not.